### PR TITLE
Calculator logic

### PR DESCRIPTION
**Pull request by ikraamg**
#### Dear TSE 
 [Heroku Link](https://math-magicians-calculator.herokuapp.com/)
 [Readme](https://github.com/ikraamg/calculator-react/blob/calculator-logic/README.md)

In this PR, I have:
- Created the operate and calculate files.
- Added basic logic into the operate file.
- Imported the Big.js into the operate.
 
Thank you for your time.

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
#### Hi @ikraamg , you submitted this milestone under milestone 4, kindly resubmit it under milestone 3 as required. I will mark it invalid for now. See you soon🤦‍♂️

**Comment by ikraamg** [comment]
Hi @Forison, this is actually milestone 4 but because I submitted both 3 and 4  before waiting for 3 to be approved it contains the commits of 3 as well. I will create a new PR and resubmit.

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+
+export default function calculate(
+  data = {
```
Comment (Forison) [inline src/logic/calculate.js:4]: ```({total ,next, operation}, buttonName )``` you can access the total, next and operation in the code afterwards
or
```(data, buttonName)``` the restructure next, total and operation in the case.
I think this way is more cleaner than what we have now

**Review by Forison — APPROVED** [review]
## Status:: approved✔️
# Hi @ikraamg 
### You have done a good job on this one so far, check my comment below to improve your code, even what you have meets the requirement, we can handle it better than you have done🤗.
#### Happy coding👨‍💻
Addo Forison

**Comment by Forison** [comment]
> Hi @Forison, this is actually milestone 4 but because I submitted both 3 and 4 before waiting for 3 to be approved it contains the commits of 3 as well. I will create a new PR and resubmit.

sorry at @ikraamg, I realised my mistake later, I marked invalid, I am sure it may be validated in the next review.

**Pull request closed without merging**